mainmatter.md

```diff
@@ -258,23 +237,21 @@ We would like to thank [...] for their valuable feedback.
 
 # IANA Considerations {#IANA}
       
-This draft includes no request to IANA.
-    
+This draft includes no request to IANA.  
```

Comment:
The current draft does include IANA requests.  I suspect that this like is currently a merge conflict.